src/lua/api-gateway/redis/redisConnectionProvider.lua

```diff
@@ -76,10 +81,18 @@ function RedisConnectionProvider:getConnection(connection_options)
     end
 
     local redisHost, redisPort = self:getRedisUpstream(redisUpstream)
-    return self:connectToRedis(redisHost, redisPort, redisPassword, redisTimeout)
+    ngx.log(ngx.WARN, "Trying with: " .. tostring(redisHost) .. " and " .. tostring(redisPort))
```

Comment:
First "try" should be logged as `DEBUG`?

Reply:
Yes, definitely. I will update now.